hack/release-helper.sh

```diff
@@ -9,6 +9,7 @@ declare -g release_tag
 declare -A staged_to_release=(
     ["staged-images/kbs"]="key-broker-service"
     ["staged-images/kbs-grpc-as"]="key-broker-service"
+    ["staged-images/kbs-ita-as"]="key-broker-service"
```

Comment:
This part remaps the image names, but I think you'll also need to create an entry below where the tag prefix is generated. Otherwise, we're going to have a collision with the image names. We should maybe add some comments to the script to make it more clear.

Reply:
Fixed along with release-guide + comment (which is helpful in this case)